src/Symfony/Component/VarDumper/Caster/AmqpCaster.php

```diff
@@ -46,7 +46,7 @@ class AmqpCaster
 
     public static function castConnection(\AMQPConnection $c, array $a, Stub $stub, $isNested)
     {
-        $prefix = "\0~\0";
+        $p = Caster::PREFIX_VIRTUAL;
```

Comment:
Can't we keep the old variable name? To me, `$prefix` is more meaningful than a one-character variable name.